go/interpreter/lfvm/instructions_test.go

```diff
@@ -1879,6 +1879,29 @@ func TestInstructions_ReturnDataCopy_ReturnsErrorOnParameterOverflow(t *testing.
 	}
 }
 
+func TestOpExtCodeHash_WritesHashInStack(t *testing.T) {
```

Comment:
This test is targeting a single missing branch of the operation, but it would have more value if it models the operation behavior with more detail.

- WritesCodeHashInStackIfAccountExists
- WritesZeroesInStackIfAccountDoesNotExist

or using cases
- TestInstructions_OpCodeHash/WritesCodeHashInStackIfAccountExists
- TestInstructions_OpCodeHash/WritesCodeHashInStackIfAccountExists